packages/paste-website/src/pages/components/table/index.mdx

```diff
@@ -57,47 +67,79 @@ export const pageQuery = graphql`
-    your experience using this component by{' '}
-    <Anchor href="https://github.com/twilio-labs/paste/discussions/new">starting a discussion on GitHub</Anchor>. We are
-    also looking for a contributor to symbolize this component in Sketch. Interested? Reach out!
-  </CalloutText>
-</Callout>
-
 ## Guidelines
 
 ### About Table
 
-A Table is used to represent tabular data, or information presented in a 2-dimensional table composed of rows and columns of cells containing data.
-Tables fill 100% width of the space of their containing element.
-
-You can create tables using a combination of:
-
-- **Table:** The table wrapper.
-- **THead:** The header of your table.
-- **TBody:** The main body of your table.
-- **TFoot:** The footer of your table.
-- **Tr:** A row of cells in your table.
-- **Th:** A header cell which can be used as a column title.
-- **Td:** A cell of table content.
+A Table is a grid of static textual and numeric data of the same content structure presented in rows and columns of cells.
+For example, a Table would be a good way to represent a list of multiple support agents and their respective skills
+and online/offline status.
 
 ### Accessibility
 
 Tables are used to organize data with a logical relationship in grids. Accessible tables need HTML markup that indicates
-header cells and data cells and defines their relationship. Assistive technologies use this information to provide context to users.
+header cells and data cells and defines their relationship. Assistive technologies use this information to provide
+context to users.
 
-In order to be fully accessible:
+Accessibility requirements:
 
-- Tables **must** have a `THead`, containing a Table Header `Th` for each column.
-- Tables **must** have a `TBody` wrapping the table body of rows.
+- Tables must have a `THead`, containing a Th for each column.
+- Tables must have a `TBody` wrapping the table body of rows.
+
+Accessibility recommendations:
+
+- It is a best practice to include row headers, which can be added by including a header (`Th`) as the first cell in
+  each row (`Tr`), with `scope="row"`. This is not relevant for all tables, but you should follow this practice when
+  the first cell in a row is the key value for that row, such as a username. You can find an example of this in
+  the section on [row headers](#row-headers).
+- Any component can be placed in a Table, but tables should only contain static textual and numeric data, rather
+  than actionable components like buttons, whenever possible. This is because, if the table is reasonably long—say
+  50 rows—with multiple actions per row—say 4 actions—a keyboard-only user would have to press the tab key 200
+  times to get past that table.
+
+<Callout>
+  <CalloutTitle>🦮 Accessibility insight: navigating tables with a screen reader</CalloutTitle>
+  <CalloutText>
+    <Anchor href="https://youtu.be/tlpQ0d2ADNo" showExternal>
+      Watch this video
```

Comment:
I'm not sure this link text is great for a11y

Reply:
Good catch @TheSisb. How about: "To learn more about why semantic table markup is so important, watch this <a>demo video by Inclusive Components</a> to see what it's like to explore a data table with a screen reader."

Reply:
@richbachman the anchor tags I put around the linked text in my comment disappeared haha sorry! I was thinking the linked text should be "this demo video by Inclusive Components".